packages/cf-component-textarea/src/TextareaTheme.js

```diff
@@ -0,0 +1,32 @@
+export default baseTheme => ({
+  height: 'initial',
+  wordBreak: 'initial',
+  cursor: 'initial',
+  resize: 'initial',
+
+  zIndex: 0,
+  width: '100%',
+  margin: '0 0 0.75rem',
+  padding: '0.45em 0.75em',
+  border: '1px solid #bdbdbd',
```

Comment:
@koddsson Would love to see a screenshot to get more context. If it's really close to another color in our variables we should replace it. If it should be unique we can add a new variable for it. Right now it's hard for me to see where/how it's being used.